package.json

```diff
@@ -33,7 +33,7 @@
     "start-examples": "ws -d examples/ -s index.html -p 4242",
     "start-docs": "docsify serve ./docs-user",
     "start-photon": "node res/photon/server",
-    "test": "cross-env LC_ALL=C NODE_ENV=test jest",
+    "test": "cross-env LC_ALL=C TZ=UTC NODE_ENV=test jest",
```

Comment:
Tested this with macOS as well, and it's working fine on my machine. Looks good!